### PR TITLE
fix: stop swap layout shifts when reverse token button clicked

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-996?atlOrigin=eyJpIjoiZDdjZTY3N2JmNTI5NDBhYmJmNThmOTJlN2UyYzJlMjMiLCJwIjoiaiJ9

to reproduce: 
- dont connect any wallet
- dont select second output token
- click reverse tokens button on swap page
- there should be no layout movements

### CODE REVIEW

File: src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ const FixedContainer = styled.div<{ redesignFlag: boolean }>`
 `
 
 const Container = styled.div<{ hideInput: boolean; disabled: boolean; redesignFlag: boolean }>`
+  min-height: ${({ redesignFlag }) => redesignFlag && '69px'};
```
Comment (kayleegeorge) [inline src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx:54]: do we want to make this a round number? either 68 or 72?

**Review by kayleegeorge — APPROVED** [review]
lgtm, commented about px num (usually want to do divisible by 4)

File: src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx:54]: 69 is the needed number to make no shifts (due to the size of the other section)

Comment (lynnshaoyu) [reply src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx:54]: also, lolz :kayleedead:

Comment (kayleegeorge) [reply src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx:54]: lmfaooo

**Pull request merged**